airbyte-webapp/src/packages/cloud/views/users/AccountSettingsView/components/NameSection/NameSection.tsx

```diff
@@ -0,0 +1,60 @@
+import { Field, FieldProps, Form, Formik } from "formik";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { LoadingButton } from "components";
+import { LabeledInput } from "components/LabeledInput";
+
+import { useCurrentUser } from "packages/cloud/services/auth/AuthService";
+import { RowFieldItem } from "packages/cloud/views/auth/components/FormComponents";
+import FeedbackBlock from "pages/SettingsPage/components/FeedbackBlock";
+import { Content, SettingsCard } from "pages/SettingsPage/pages/SettingsComponents";
+
+import { useChangeName } from "./hooks";
+import { useValidation } from "./validation";
+
+export const NameSection: React.FC = () => {
+  const validate = useValidation();
+  const { formatMessage } = useIntl();
+  const user = useCurrentUser();
+  const { changeName, successMessage, errorMessage } = useChangeName();
+
+  return (
+    <SettingsCard title={<FormattedMessage id="settings.account" />}>
+      <Content>
+        <Formik
+          initialValues={{
+            name: user.name,
+          }}
+          onSubmit={changeName}
+          validate={validate}
```

Comment:
We've been using `yup` to validate the schema. Search the code for an example.

Reply:
updated